### DISCUSSION TITLE
Is it possible to set the initial state manually?

### DISCUSSION THREAD

**Discussion by hyjforesight** [theislab/cellrank#786] [Q&A] [answered]
Hello CellRank,
When running the low-level mode using GPCCA, the terminal state can be mannually by estimator.set_terminal_states_from_macrostates() or estimator.set_terminal_states(). But GPCCA cannot set the initial state. Could you please tell us whether there is a way to set the initial state mannually in CellRank?
Thanks!
Best,
YJ

**Comment by hyjforesight** [comment, marked as answer]
link to https://github.com/theislab/cellrank/issues/787

**Reply by Marius1311** [reply]
This depends on what application you have in mind - we implemented the functionality to manually set terminal states so that they can be used downstream for computing fate probabilities. How would you like to use the manually annotated initial states in downstream computations?

**Reply by Marius1311** [reply]
Sorry, didn't see this was answered in the linked issue already.
